doc/developer/build_release.rst

```diff
@@ -88,25 +88,27 @@ Use the ``ngeo`` package linked to the new branch
 In ``c2cgeoportal`` new version branch, in the file ``geoportal/package.json``, set the ``ngeo`` version to
 ``version-<new version>-latest``.
 
-Create the new transifex resources
+Create the new Transifex resources
 ----------------------------------
 
 Run:
 
 .. prompt:: bash
 
-    docker build --target=tools --tag=transifex --build-arg=MAJOR_VERSION=x.y .
-    docker run --name=transifex -ti --rm --detach --volume=${HOME}:/root transifex tail -f /dev/null
-    docker exec transifex bash -c \
-        '(cd /opt/c2cgeoportal; sed -i s/x_y/x_y+1/g .tx/config)'
-    docker exec --env=MAJOR_VERSION=x.y+1 transifex bash -c \
-        '(cd /opt/c2cgeoportal; make --makefile=dependencies.mk transifex-init)'
-    docker stop transifex
+    tx pull --branch=<version> --source --force \
```

Comment:
can this be executed on host, shouldn't we use tools container to run those commands

Reply:
Yes, we can we just need to have the Transifex client and be logged in, but it's true that it we do that in Docker it's possible that we have less issues...